docs/source/guide/benchmarks.md

```diff
@@ -0,0 +1,138 @@
+        .real
+    )
+
+circuit = benchmarks.generate_ghz_circuit(n_qubits=7) # Call the required benchmark circuit function here
+
+true_value = execute(circuit, noise_level=0.0)      # Ideal quantum computer
+noisy_value = execute(circuit)                      # Noisy quantum computer
+zne_value = zne.execute_with_zne(circuit, execute)  # Noisy quantum computer + Mitiq
+
+print(f"Error w/o  Mitiq: {abs((true_value - noisy_value) / true_value):.3f}")
+print(f"Error w Mitiq:    {abs((true_value - zne_value) / true_value):.3f}")
+```
+
+
+## GHZ Circuits
+
+The {func}`.generate_ghz_circuit` create the GHZ states that are highly sensitive to noise. Thus, they make it easy to test error rates in entanglement creation and preservation, which is central for many quantum algorithms.
+
+```{code-cell} ipython3
+from mitiq.benchmarks import generate_ghz_circuit
+
+circuit = generate_ghz_circuit(n_qubits=7)
+print(circuit)
+```
+
+## Mirror Circuits
+
+The {func}`.generate_mirror_circuit` involves running a quantum circuit forward and then “mirroring” it (applying the reverse operations). Ideally, this results in returning the system to the initial state, so they’re great for testing if the noise mitigation is effective in preserving information through complex sequences.
+
+## Mirror Quantum Volume Circuits
+
+The {func}`.generate_mirror_qv_circuit` is designed to test `quantum volume`, a metric combining circuit depth, number of qubits, and fidelity. These circuits check whether error mitigation techniques help achieve higher effective quantum volumes on noisy devices.
+
+```{code-cell} ipython3
+from mitiq.benchmarks import generate_mirror_qv_circuit
+
+circuit= generate_mirror_qv_circuit(num_qubits=7, depth=2)
+print(circuit)
+```
+
+## Quantum Phase Estimation Circuits
+
+The {func}`.generate_qpe_circuit` is used to the measure eigenvalues of unitary operators. Since accurate phase estimation requires precise control over operations, these circuits test the mitigation techniques’ ability to handle small noise effects over multiple gate sequences.
+
+```{code-cell} ipython3
+from mitiq.benchmarks import generate_w_circuit
+
+circuit = generate_w_circuit(n_qubits=7)
+print(circuit)
+```
+
+## Quantum Volume Circuits
+
+The {func}`.generate_quantum_volume_circuit` tests the maximum achievable "volume" or computational capacity of a quantum processor. Running these circuits with error mitigation tests if mitiq’s techniques improve the effective quantum volume.
+
+```{code-cell} ipython3
+from mitiq.benchmarks import generate_quantum_volume_circuit
+
+circuit,_ = generate_quantum_volume_circuit(num_qubits=4, depth=7)
+print(circuit)
+```
+
+## Randomized Benchmarking Circuits
+
+The {func}`.generate_rb_circuits` are sequences of random gates (generally Clifford gates), to estimate an average error rate. They’re standard in benchmarking for evaluating how well mitiq’s error mitigation reduces this error rate across different levels of noise.
```

Comment:
Can you add some details about what the state of each circuit is supposed to be? For example, the ideal output of a randomized benchmarking circuit is an identity. 

GHZ and W circuits define a state. So, you could use some latex equations to show what the states are supposed to be. 

We don't want to describe what the addition of noise will do to a circuit. the focus is on the details of the benchmarking circuit. All docstrings link a reference used to define a particular benchmarking circuit. You could use details from these refs if you don't know a lot about these circuits.

Reply:
You can also use [`.final_state_vector()`](https://quantumai.google/reference/python/cirq/final_state_vector) to show the ideal circuit state, which is what we expect it to be. 

```python
from mitiq.benchmarks import generate_ghz_circuit
circuit = generate_ghz_circuit(n_qubits=3)
circuit.final_state_vector()
```
this will spit out the 1-D vector form of a 3 qubit GHZ state. 

![image](https://github.com/user-attachments/assets/0d0f16a4-481b-4fe1-b90f-2d23ddfdd6e8)

Reply:
> You can also use [`.final_state_vector()`](https://quantumai.google/reference/python/cirq/final_state_vector) to show the ideal circuit state, which is what we expect it to be.

Yes, I can add this function to the example workflow at the top of the file, so every circuit's ideal circuit state can be seen.

Reply:
> We don't want to describe what the addition of noise will do to a circuit. the focus is on the details of the benchmarking circuit. All docstrings link a reference used to define a particular benchmarking circuit. You could use details from these refs if you don't know a lot about these circuits.

Since the section is about benchmarking, I believe we should show improvements with mitiq when compared to a noisy circuit without mitiq. But if you think otherwise, I can remove that part from the examples.

Also, since I have now added [.final_state_vector()](https://quantumai.google/reference/python/cirq/final_state_vector), shall I still go ahead and add the latex equations to show what the states are supposed to be?

Reply:
The focus is on benchmarking circuits. Not benchmarking the performance of a QEM technique using mitiq.